ZenPacks/zenoss/Microsoft/Windows/zenpack.yaml

```diff
@@ -47,7 +47,6 @@ zProperties:
     default: false
 
 class_relationships:
-  - ZenPacks.zenoss.Microsoft.Windows.OperatingSystem.OperatingSystem(winrmservices) 1:MC (os)WinService
```

Comment:
I don't think you can do this. You'll certainly end up in a situation where the relationship schema doesn't match the relationship reality. It's not so bad when the schema contains relationships that an object doesn't, but it's very bad when the object contains a relationship that doesn't exist in the schema.

You need to leave vestigial relationships around forever. You can empty them, but they need to remain.